### PR TITLE
add style target to makefile

### PR DESCRIPTION
**Pull request by johko**
Added the `style`  target to the makefile, so you actually can run `make style` to fix formatting. 
(command shamelessly copied from the audio course https://github.com/huggingface/audio-transformers-course/blob/main/Makefile )

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,4 +2,8 @@
```
Comment (jungnerd) [inline Makefile:2]:
```suggestion
.PHONY: quality style
```
On line 1, I think maybe `style` should be added here.

File: Makefile [hunk 2]
```diff
@@ -2,4 +2,8 @@
 
 # Check code formatting
 quality:
-	python utils/code_formatter.py --check_only
\ No newline at end of file
+	python utils/code_formatter.py --check_only
+
+# Format code samples automatically and check is there are any problems left that need manual fixing
```
Comment (jungnerd) [inline Makefile:7]:
```suggestion
# Format code samples automatically and check if there are any problems left that need manual fixing
```
Fixed a typo.

**Review by jungnerd — COMMENTED** [review]
I've reviewed it. Please check it out 🤗

**Comment by johko** [comment]
Thanks for the suggestions @jungnerd, both make sense to me :hugs:

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,9 @@
 .PHONY: quality
-
+.PHONY: quality style
```
Comment (sergiopaniego) [inline Makefile:2]: Could this line be merged with line 1 as in https://github.com/huggingface/audio-transformers-course/blob/main/Makefile?

**Review by merveenoyan — APPROVED** [review]
Thanks!

File: Makefile (hunk 3 above)
Comment (johko) [reply Makefile:2]: good point, I thought that was what it did when I accepted the proposed change, but seems like something was going a bit wrong there

**Pull request merged**